src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2479,6 +2506,22 @@ export class StudyViewPageStore {
         default: []
     });
 
+    readonly fusionGeneData = remoteData<FusionGenesData>({
+        await: () => [this.mutationProfiles],
+        invoke: async () => {
+            if (!_.isEmpty(this.mutationProfiles.result!)) {
+                // TODO: get data for all profiles
```

Comment:
not sure if i understand the TODO

Reply:
sorry, I forgot too. It has been really a while, will back to you later.

Reply:
I will remove these TODOs. I placed there as a reminder for a feature which should be transferred to a proper issue on github.